helm/cert-exporter/Chart.yaml

```diff
@@ -7,4 +7,5 @@ home: https://github.com/giantswarm/cert-exporter
 version: 2.1.0
 appVersion: 2.0.0
 annotations:
+  application.giantswarm.io/team: "hydra"
```

Comment:
@alex-dabija should we own `cert-exporter`?

Reply:
No we shouldn't. According to the latest updates, which might not have been propagated yet to the repos and the intranet ownership page, it's actuall @giantswarm/team-phoenix which owns cert-exporter. See [doc](https://docs.google.com/spreadsheets/d/1GNXAbYo_HyjjRusd2KTGttuORBkX9iMdgnNp-xTQ9sw/edit#gid=0).

Reply:
we appreciate and support any contributions whatsoever or claiming ownership:trollface:

Reply:
Well, it sparked the discussion :smile: Fixed now :)